roles/openshift_health_checker/openshift_checks/docker_image_availability.py

```diff
@@ -0,0 +1,49 @@
+# pylint: disable=missing-docstring
+from openshift_checks import OpenShiftCheck, get_var
+from openshift_checks.mixins import NotContainerized
+
+class DockerImageAvailability(NotContainerized, OpenShiftCheck):
+    """Check that required Docker images are available."""
+
+    name = "docker_image_availability"
+    tags = ["preflight"]
+
+    def run(self, tmp, task_vars):
+        openshift_release = get_var(task_vars, "openshift_release")
+
+        args = {
+            "name": self.rpm_docker_images(),
```

Comment:
@rhcarvalho what do you think of instead having a method like `docker_images_for()` that takes a `deployment_type` like `origin` or `openshift-enterprise` and then a `version`?  That method could contain the logic to determining if a image uses tags like `v3.4` or `v3.4.1.40`.